### PR TITLE
Adds guide on how to build a custom CL container image.

### PR DESCRIPTION
**Pull request by iphutch**
Signed-off-by: ihutchin <ianeta.hutchinson@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a |CL|-based container
+----------------------------------------
+#. To add a bundle to an existing |CL|-based container, use the "`swupd
```
Comment (tpleavitt) [inline source/clear-linux/guides/network/custom-clear-container.rst]: Why is every line after this underlined?

Comment (tpleavitt) [reply source/clear-linux/guides/network/custom-clear-container.rst]: @iphutch , @rcaballeromx , do you see these artifacts?

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 2]
```diff
@@ -0,0 +1,369 @@
+.. _custom-clear-container:
+
+Build a Custom |CL|-based Docker Container Image
```
Comment (iphutch) [inline source/clear-linux/guides/network/custom-clear-container.rst]: Headings/titles should be in sentence case. The |CL| substitution doesn't work in headings. Let's avoid a hyphen following Clear Linux
- Have only the first letter of: the sentence, Clear Linux, and Docker be capitalized.
- Write out the full "Clear Linux\\*" and drop the hyphen.
- Docker\\* (the \\ ensures the * doesn't make everything enclosed italic)

Build a custom Clear Linux\\* based Docker\\* container image

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+          ---> ec23189ef954
+         Removing intermediate container 7694989e97de
+         Successfully built ec23189ef954
+         Successfully tagged my-clearlinux-remove-pxe-server-bundle:latest
+
+Also see:
+
+   * :ref:`cc-getting-started`
+
+.. _bundles: https://clearlinux.org/documentation/clear-linux/reference/bundles/available-bundles.html#bundle-list
```
Comment (iphutch) [inline source/clear-linux/guides/network/custom-clear-container.rst]: remove as we will use the ref reference

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a |CL|-based container
+----------------------------------------
+#. To add a bundle to an existing |CL|-based container, use the "`swupd 
+   bundle-add`" command.  Here’s an example Dockerfile that shows adding the
```
Comment (iphutch) [inline source/clear-linux/guides/network/custom-clear-container.rst]: - remove all quotation marks. 
- s/ |CL|-based/ |CL| based on line 260
- s/ Here's/ Here is
- remove whitespace after "the"

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+         my-custom-clear-linux-container   latest              5414c3a12993        About a minute ago   616MB
+
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a |CL|-based container
+----------------------------------------
```
Comment (iphutch) [inline source/clear-linux/guides/network/custom-clear-container.rst]: - Write out the full "Clear Linux"
- Drop the hyphen before "based"
- Upgrade the heading syntax
- Add a blank line after the heading syntax

**Comment by iphutch** [comment]
@bktan8, changes applied. 
@rcaballeromx this PR is ready for your review.

File: source/clear-linux/guides/network/custom-clear-container.rst [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+#. Launch the built |CL| container.
+
+   .. code-block:: console
+
+      # docker run -it my-custom-clear-linux-container
+
+Manage bundles in a Clear Linux based container
+***********************************************
+
+#. To add a bundle to an existing |CL|-based container, use the
```
Comment (rcaballeromx) [inline source/clear-linux/guides/network/custom-clear-container.rst]: These should not be numbered steps. Rather subsections: Add a bundle and Remove a bundle with the numbered steps on how to do those actions and verify them. Avoid just mentioning actions to be taken or that have been taken and be explicit about how they were performed.

**Comment by iphutch** [comment]
@rcaballeromx when you are reviewing the rest of the doc, please ignore the new vnc file. In the next commit to address the remainder of your review, we will push a clean PR.

**Comment by rcaballeromx** [comment]
Understood @iphutch. I'll try to get it done today.

**Pull request closed without merging**